scripts/autoformat_jsonnet.sh

```diff
@@ -18,10 +18,22 @@
 # Assumes jsonnet is on the path.
 set -ex
 
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
What does script_dir end up being set to? Why isn't using pwd sufficient?

I think we want to use the current working directory because we want to be able to use this script to format jsonnet files in other repositories.

Reply:
script_dir is the directory in which `autoformat_jsonnet.sh` lives. 

pwd also works fine - the reason I made this change was to support calling `autoformat_jsonnet.sh` from any directory. The result would be same whether the script was executed from the root directory of the repo or the `scripts/` directory of the repo.

I can change it back to pwd if you prefer.